release/README.md

```diff
@@ -8,22 +8,55 @@ Template configurtion file for the scripts found in this directory.
 Copy to ~/.server.config and fill out with the appropriate values for
 services provided on the server running these automation scripts.
 
+---
+
+## Automated Publishing Scripts
+
+The `crontab` configuration driving this automation:
+```
+03 */2 * * * bin/release/mk-release-snapshots.sh
+21 */2 * * * bin/release/mk-cfgman-docs.sh
+23 */2 * * * bin/release/mk-langpack.sh
+25 */2 * * * bin/release/mk-www-manuals.sh
+```
+
 ### mk-cfgman-docs.sh
 
 Publish the latest [Squid Configuration Guide](http://www.squid-cache.org/Doc/config/).
 
+Fully automated ROLLING release.
+
 ### mk-langpack.sh
 
 Publish the latest [Squid Translation Package](http://www.squid-cache.org/Versions/langpack/).
 
+Fully automated ROLLING release.
+
 ### mk-release-series.sh
 
 Branch and publish the next [Squid Series](https://wiki.squid-cache.org/ReleaseSchedule).
 
+This script serves two roles:
+ 1. a checklist for release manager(s) on tasks required to branch a new major release series of Squid.
+ 2. a script to ease those tasks as much as possible.
+
+This script is intended to be run manually with review of the changes is performed, and additional steps are required for the complete process.
+
```

Comment:
Feels wrong to add this information here. IMO, given the two possibly locations, it should be inside the script itself, where it has more chances of getting updated (as the script changes) and where it can benefit from some script-specific context.

Reply:
These are the highest level abstraction description of the script purpose. IMO the text actually inside should be a fair bit more detailed/specific than this.

Unless you happen to know a way to import the documentation comment at the top of these scripts into this file display using Markdown. In which case we could put everything inside them.

Reply:
> These are the highest level abstraction description of the script purpose

I disagree. The existing description was the highest level of abstraction. The proposed text is mostly documenting current (and, IMHO, unfortunate) implementation choices.

> Unless you happen to know a way to import the documentation comment at the top of these scripts into this file display using Markdown

Well, we can obviously generate this README, but I am _not_ suggesting that kind of complication at this point.

Reply:
Very well, to avoid further arguments removing documentation about this script